src/AppInstallerCLICore/Commands/SettingsCommand.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "SettingsCommand.h"
+#include "Workflows/WorkflowBase.h"
+#include "Resources.h"
+#include <winget/UserSettings.h>
+
+namespace AppInstaller::CLI
+{
+    using namespace Utility::literals;
+    using namespace AppInstaller::Settings;
+    using namespace AppInstaller::Utility;
+
+    using namespace std::string_view_literals;
+
+    std::vector<Argument> SettingsCommand::GetArguments() const
+    {
+        return {};
+    }
+
+    Resource::LocString SettingsCommand::ShortDescription() const
+    {
+        return { Resource::String::SettingsCommandShortDescription };
+    }
+
+    Resource::LocString SettingsCommand::LongDescription() const
+    {
+        return { Resource::String::SettingsCommandLongDescription };
+    }
+
+    std::string SettingsCommand::HelpLink() const
+    {
+        return "https://aka.ms/winget-settings";
+    }
+
+    void SettingsCommand::ExecuteInternal(Execution::Context& context) const
+    {
+        // Show warnings only when the setting command is executed.
+        if (!User().GetWarnings().empty())
+        {
+            context.Reporter.Warn() << Resource::String::SettingLoadFailure << std::endl;
+            for (const auto& warning : User().GetWarnings())
+            {
+                context.Reporter.Warn() << warning << std::endl;
+            }
+        }
+
+        User().PrepareToShellExecuteFile();
+
+        
+        auto filePathUTF16 = ConvertToUTF16(UserSettings::SettingsFilePath().u8string());
```

Comment:
>ConvertToUTF16(UserSettings::SettingsFilePath().u8string()); [](start = 29, length = 60)

UserSettings::SettingsFilePath().c_str() [I think... the point is there is a method that just gives back the wide version]